### PR TITLE
[DO NOT MERGE] Threadlocal and additional setup

### PR DESCRIPTION
**Pull request by lanking520**
HI @skirdey I got it run on my Mac with some addtional setup. Here is some recomended changes. Since the TensorFlow predictor is not threadsafe, using threadlocal can help to avoid segfaults. Another issue I observe is the problem with the `maxphysicalbytes`, I hit into this once I tried to run several inference call. I think it is a bug design in JavaCPP to calculate the maximum memory space.

@frankfliu

### CODE REVIEW

File: src/main/scala/djlakkahttpexample/UseExample.java [hunk 1]
```diff
@@ -22,15 +22,11 @@
 
 public final class UseExample {
 
+    static final ZooModel<String[], float[][]> model = loadModel();
+    static final ThreadLocal<Predictor<String[], float[][]>> predictorHolder = new ThreadLocal<>();
```
Comment (roywei) [inline src/main/scala/djlakkahttpexample/UseExample.java:26]: TF predictors are thread safe. It use TFSymboBlock and TensorFlow Session under the hood. They are thread safe now. Eager Sessions are not thread safe, eager sessions are used in Translator, each translator will create it's own sub manager for input data. Each submanager will have its own eager session. So it's fine. We limit the number of thread eager session can create in TF_NUM_INTEROP_THREADS  and TF_NUM_INTRAOP_THREADS

**Review by roywei — APPROVED** [review]
LGTM, need to test whether we need thread local predictor. Maybe we can avoid it.

File: src/main/scala/djlakkahttpexample/UseExample.java (hunk 1 above)
Comment (skirdey) [reply src/main/scala/djlakkahttpexample/UseExample.java:26]: Good to know!

File: src/main/scala/djlakkahttpexample/QuickstartApp.scala [hunk 2]
```diff
@@ -30,6 +30,8 @@ object QuickstartApp {
   }
   //#start-http-server
   def main(args: Array[String]): Unit = {
+    System.setProperty("org.bytedeco.javacpp.maxphysicalbytes", "0") // override to avoid memory overflow
```
Comment (skirdey) [inline src/main/scala/djlakkahttpexample/QuickstartApp.scala:33]: i saw this issue when running locally, but haven't seen it in a deployed service :''\_(-_-)_/```

**Review by skirdey — APPROVED** [review]

**Pull request merged**